gxformat2/export.py

```diff
@@ -56,6 +59,9 @@ def from_galaxy_native(native_workflow_dict, tool_interface=None, json_wrapper=F
 
     # For each step, rebuild the form and encode the state
     for step in native_steps.values():
+        position = prune_position(step)
```

Comment:
Having feature parity between .ga and Format 2 as the goal, can we confidently drop the other `position` fields (bottom, height, right, width, x, y) that .ga files contain for subworkflows?

Reply:
Did I miss a spot for subworkflows, or is the question if it's safe to drop the the remaining position elements ? Or that we should drop them from editor export altogether ?

Reply:
> Did I miss a spot for subworkflows

It seems that .ga files have the extra `position` fields only for subworkflows, not tools or inputs.

> or is the question if it's safe to drop the the remaining position elements ? Or that we should drop them from editor export altogether?

My question is if these fields (bottom, height, right, width, x, y) are used by Galaxy, so if we drop them when converting to Format 2 here we are losing information (and so feature parity between the 2 formats).

Reply:
> It seems that .ga files have the extra `position` fields only for subworkflows, not tools or inputs.
> 
> >

I haven't seen this, the position field should be independent of the step type.




> My question is if these fields (bottom, height, right, width, x, y) are used by Galaxy, so if we drop them when converting to Format 2 here we are losing information (and so feature parity between the 2 formats).

Re-upload maintains the coordinates, but I've asked @jmchilton and @guerler for confirmation.

Reply:
> I haven't seen this, the position field should be independent of the step type.
> 
> >

https://github.com/galaxyproject/SARS-CoV-2/blob/master/genomics/deploy/workflows/Genomics-4-All-in-one-subworkflow.ga has a mix of both styles, but it doesn't seem to depend on whether it's subworkflow or not. It might have to do on which version of Galaxy the workflow was defined ?

Reply:
@guerler confirmed that we only need left and top. I think that we store the the other values is probably a bug on the Galaxy side.